analyses/oncoprint-landscape/run-oncoprint.sh

```diff
@@ -15,26 +15,28 @@ cd "$script_directory" || exit
 
 #### Download the driver lists from the PPTC repository
 
-mkdir -p driver-lists
-wget -N --directory-prefix=driver-lists -O driver-lists/brain-goi-list-long.txt https://github.com/marislab/create-pptc-pdx-oncoprints/raw/2c9ed2a2331bcef3003d6aa25a130485d76a3535/data/brain-goi-list-old.txt
-wget -N --directory-prefix=driver-lists -O driver-lists/brain-goi-list-short.txt https://github.com/marislab/create-pptc-pdx-oncoprints/raw/2c9ed2a2331bcef3003d6aa25a130485d76a3535/data/brain-goi-list.txt
+# mkdir -p driver-lists
```

Comment:
I commented it out for development reasons mainly. We are using the gene list from the `interaction-plots` directory in this PR instead of these driver gene lists so this is technically not needed here unless we want to leave the option in to implement them instead.